lib/holdings/item.rb

```diff
@@ -214,7 +214,7 @@ def folio_item?
     private
 
     def internet_resource?
-      home_location == 'INTERNET'
+      home_location == 'INTERNET' || type == 'ONLINE'
```

Comment:
Can you add the comment here about this being for Lane resources?

Reply:
I'm not sure why that's appropriate; it's not just for Lane resources (although because none of Lane's eresources have the home location INTERNET, it's easy to find examples)